Fw/Python/src/fprime/fbuild/builder.py

```diff
@@ -580,52 +575,47 @@ class AmbiguousToolchainException(FprimeException):
     LocalTarget(
         "build",
         "Build unit tests",
-        build_types=[BuildType.BUILD_TESTING],
```

Comment:
Are these targets no longer supported on multiple build types?

Reply:
If you saw my explanation above, this was the compromise I made. Targets supporting multiple build types caused issues cause they wouldn't list properly in `fprime-util info` and implicitly setting the build type to release required users to manually pass in `--ut` in order to run targets like `fprime-util check`.

Reply:
So what happens (eventually) when we run build --debug?  Would this not allow us to build something using the `debug`?